### PR TITLE
Clarify the description of adding icons to templates

### PR DESCRIPTION
**Pull request by Ibochkarev**
### What does it do?
Clarify the description of adding icons to templates

### Why is it needed?
The user was initially unclear what icons are used and in what form they should be indicated in the field for entering Icons. This PR improves the description.

### Related issue(s)/PR(s)
NA

### CODE REVIEW

File: core/lexicon/en/template.inc.php [hunk 1]
```diff
@@ -33,7 +33,7 @@
 $_lang['template_err_remove'] = 'An error occurred while removing the template.';
 $_lang['template_err_save'] = 'An error occurred while saving the template.';
 $_lang['template_icon'] = 'Icon';
-$_lang['template_icon_description'] = 'Optional. A custom icon class for all resources with this template.';
+$_lang['template_icon_description'] = 'Optional. A custom Font Awesome (Free) icon class for all resources with this template. For example: icon-home or fa-home.';
```
Comment (Jako) [inline core/lexicon/en/template.inc.php]: `css icon class` instead of `Font Awesome (free) icon class`?

Comment (Ibochkarev) [reply core/lexicon/en/template.inc.php]: I am change.

Comment (Ruslan-Aleev) [reply core/lexicon/en/template.inc.php]: @Jako Without clarifying Font Awesome, it’s not obvious which class to use, especially for new users.

Comment (Ibochkarev) [reply core/lexicon/en/template.inc.php]: @Jako I agree with Ruslan. This description was born during the discussion.

Comment (Jako) [reply core/lexicon/en/template.inc.php]: Then write `(i.e. a Font Awesome class)`, since that part depends on the css of the manager.

Comment (JoshuaLuckers) [reply core/lexicon/en/template.inc.php]:
```suggestion
$_lang['template_icon_description'] = 'Optional. A custom CSS icon class for all resources with this template. In example a Font Awesome (free) class such as icon-home or fa-home.';
```

**Review by JoshuaLuckers — DISMISSED** [review]

**Review by Jako — DISMISSED** [review]

File: core/lexicon/en/template.inc.php [hunk 2]
```diff
@@ -33,7 +33,7 @@
 $_lang['template_err_remove'] = 'An error occurred while removing the template.';
 $_lang['template_err_save'] = 'An error occurred while saving the template.';
 $_lang['template_icon'] = 'Icon';
-$_lang['template_icon_description'] = 'Optional. A custom icon class for all resources with this template.';
+$_lang['template_icon_description'] = 'Optional. A custom CSS icon class for all resources with this template. In example a Font Awesome (free) class such as icon-home or fa-home.';
```
Comment (Mark-H) [inline core/lexicon/en/template.inc.php]: I think "In example" should be "For example".

**Review by JoshuaLuckers — APPROVED** [review]

**Review by Mark-H — APPROVED** [review]

**Pull request merged**